h/views/activity.py

```diff
@@ -107,9 +107,9 @@ def search(self):
 
         result['opts'] = {'search_groupname': self.group.name}
 
-        # If the group has a concept of members (aka joinable_by is not None)
-        # and the user is not in the list of members, return without extra info.
-        if self.group.joinable_by and (self.request.user not in self.group.members):
+        # If the group has a concept of members and the user is not in the list of
+        # members, return without extra info.
+        if self.group.members and (self.request.user not in self.group.members):
```

Comment:
I actually think there's a problem with the logic here, @sheetaluk . If the group _does_ have some members, but the user isn't one of those members, then it returns without any info for the sidebar. This means that for example if I'm on the activity page of a restricted or open group, and I'm not a member of that group, then I don't get any group sidebar (but if I was a member then I would get a sidebar).

I'm a little bit confused as to why the `return` here was here in the first place to be honest